src/resources/views/base/inc/scripts.blade.php

```diff
@@ -62,4 +62,21 @@
     $('.nav-tabs a').on('shown.bs.tab', function (e) {
         location.hash = e.target.hash.replace("#tab_", "#");
     });
-</script>
+
+    {{-- Error frame --}}
+    @if(config('app.debug'))
+    $(document).ajaxComplete(function(e, result, settings) {
+        if(result.responseJSON?.exception !== undefined) {
+            settings.accepts = "text/html";
+            $.ajax(settings);
+        } else if(!result.responseText?.startsWith("{")) {
+            Noty.closeAll();
+            const errorFrame = document.querySelector('.error-frame');
+            errorFrame.classList.add('active');
+            errorFrame.querySelector('iframe').srcdoc = result.responseText;
+            errorFrame.querySelectorAll('.close, .background').forEach(e => e.onclick = () => errorFrame.classList.remove('active'));
+        }
+    });
+    @endif
```

Comment:
Let's move this to the `error_frame` blade file, in its own `<script>` block.

Reply:
Did it 👌